lib/transactions.js

```diff
@@ -242,55 +254,59 @@ exports.createGeneration = (rpcData, blockReward, feeReward, recipients, poolAdd
                 // console.log(`foundersAddr:  ${coin.vFoundersRewardAddress[index]}`)
 
                 // pool t-addr
-                tx.addOutput(
+                txb.addOutput(
                     scriptCompile(poolAddrHash),
                     Math.round(blockReward.total * (1 - (coin.percentFoundersReward + feePercent) / 100)) + feeReward - masternodeReward
                 )
 
                 // founders t-addr
-                tx.addOutput(
+                txb.addOutput(
                     scriptFoundersCompile(foundersAddrHash),
                     Math.round(blockReward.total * (coin.percentFoundersReward / 100))
                 )
 
                 //masternode reward
-                tx.addOutput(
+                txb.addOutput(
                     scriptCompile(masternodeAddrHash),
                     Math.round(masternodeReward)
                 )
             }
         // no founders rewards :)
         } else {
             // pool t-addr
-            tx.addOutput(
+            txb.addOutput(
                 scriptCompile(poolAddrHash),
                 Math.round(blockReward.total * (1 - (feePercent / 100))) + feeReward - masternodeReward
             )
 
             //masternode reward
-            tx.addOutput(
+            txb.addOutput(
                 scriptCompile(masternodeAddrHash),
                 Math.round(masternodeReward)
             )
         }
     }
 
-    // pool fee recipients t-addr
-    //recipients.forEach(recipient => console.log(`recipient address: ${recipient.address}`));
-    if (recipients.length > 0 && recipients[0].address != '')
-    {
-        recipients.forEach(recipient => tx.addOutput(
-            scriptCompile(bitcoin.address.fromBase58Check(recipient.address).hash),
-            Math.round(blockReward.total * (recipient.percent / 100))
-        ))
-    }
-
-    // Segwit support
+   // Segwit support
     if (rpcData.default_witness_commitment !== undefined) {
         tx.addOutput(new Buffer(rpcData.default_witness_commitment, 'hex'), 0);
     }
-
+			
+    // pool fee recipients t-addr
+    //let burn = 0
```

Comment:
I do not know what this does exactly. I have commented out until we understand what this does.